core/src/main/scala/org/apache/spark/deploy/security/ServiceCredentialProvider.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark.deploy.yarn.security
+package org.apache.spark.deploy.security
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.security.{Credentials, UserGroupInformation}
```

Comment:
I'm not quite sure this is correct though.  As it is, if a user has a service listed in the resources entry for both `deploy.security.ServiceCredentialProvider` and `deploy.yarn.security.ServiceCredentialProvider`, then they will fetch tokens twice for each service.  This should only occur if the user's workflow is to drop a duplicated, but modified version of the `deploy.yarn.security.ServiceCredentialProvider` resources file into META-INF.  If they have a fork of Spark that they merge, or if they have an entry in some other jar for just their customized service, there should be no duplication problem.

Thoughts?